### DISCUSSION TITLE
SVG logo/icon

### DISCUSSION THREAD

**Discussion by sisp** [go-task/task#720] [General]
Is there also an SVG version of the logo/icon available? I could only find a PNG image in the docs.

In addition, have you thought of a concept for a monochromatic version of the logo/icon? I.e. only a single fill and/or stroke color.

The reason I'm asking is that I'd like to add Task's icon to the [simpleicons.org](https://simpleicons.org) project, and ideally a monochromatic SVG icon already exists, a color SVG would be okay, too.

**Comment by andreynering** [comment]
Hi @sisp,

I just commited the SVG version of the logo here: https://github.com/go-task/task/blob/master/docs/Logo.svg

But regarding a monochromatic version, no, we don't have one.

**Reply by sisp** [reply]
Fantastic, thanks!

I'll prepare a few alternatives for a monochromatic version and ask for your feedback, if you don't mind. That way, Task gets an official monochromatic version of the icon and I can use it for Simple Icons.

**Reply by andreynering** [reply]
I would love to give feedback on that!

**Reply by sisp** [reply]
Here are 3 ideas for monochromatic icons:

1. Filled polygons with gaps (width has increased to 317):

   <img src="https://user-images.githubusercontent.com/2206639/167863568-60fb8dd3-6eac-49f7-9b61-eafcc081a43c.svg" width="200"/>

1. Polygon strokes (width has increased to 315.2):

   <img src="https://user-images.githubusercontent.com/2206639/167863819-99930199-6bd3-4d14-9d96-b97efe100447.svg" width="200"/>

1. Mix of filled polygon and polygon strokes (width has increased to 315.2):

   <img src="https://user-images.githubusercontent.com/2206639/167863899-6f3045f5-1afa-4a73-8b1d-7bd3e84de072.svg" width="200"/>

AFAIK, there is no way to preserve both width and height for this geometry with the typical conversion techniques (gaps, strokes) from a color to a monochromatic icon.

What do you think? By the way, feel free to dislike them all. :laughing:

**Reply by andreynering** [reply]
Tried them colored to see how it looks:

1. https://jsfiddle.net/andreynering/8hu1wakq/1/
2. https://jsfiddle.net/andreynering/tjr7kpg6/1/
3. https://jsfiddle.net/andreynering/s9f6mque/2/

I really like the first, looks very similar to the original:

![Screen Shot 2022-05-11 at 10 59 33](https://user-images.githubusercontent.com/7011819/167867922-9243b5b6-978e-4387-8e14-cdba8af87657.png)

**Reply by andreynering** [reply]
The 2nd looks better when in black, though.

**Reply by sisp** [reply]
Nice, LGTM! :tada: Do you have any concerns about the change in width?

**Reply by andreynering** [reply]
After some thought, I'd suggest you to publish the first as it looks the best when colored, and good enough when black. A good balance to me. The 2nd don't looks so right when colored.

The width looks good as is to me.

**Reply by sisp** [reply]
Alright, will do. :slightly_smiling_face:
